CHANGELOG.md

```diff
@@ -12,6 +12,13 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 - Added the ability to set `PriorityClass` through `ScanSetting`. All the scanner pods
   will be launched using the `PriorityClass` if set. This is an optional attribute, the
   scan pods will be launched without `PriorityClass` if not set.
+- The `ScanSetting` Custom Resource now allows to override the default
+  CPU and memory limits of scanner pods through the `scanLimits` attribute.
+  Please refer to the [Kubernetes documentation](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/)
+  for syntax reference.
+  In order to set the memory limits of the operator itself, modify the
+  `Subscription` object, if installed through OLM or the operator deployment
+  itself otherwise.
```

Comment:
As an end user - when do I want to modify this attribute? What are the trade-offs of using higher and lower limits?

That might be a good thing to add here if we can distill it down to a sentence or two. Otherwise, we could add this to official documentation somewhere. Perhaps in the usage guide.

Reply:
I added a sentence, does it read better?